base_multi_company/models/res_company_assignment.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+from odoo import api, fields, models, tools
+
+
+class ResCompanyAssignment(models.Model):
+
+    _name = 'res.company.assignment'
+    _auto = False
+
+    name = fields.Char()
+
+    @api.model_cr
+    def init(self):
+        tools.drop_view_if_exists(self.env.cr, self._table)
```

Comment:
@lasley The SQL view should never be dropped otherwise the foreign key into the Many2many field from `multi.company.abstract` will be set to null... '